bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyBindingConstants.java

```diff
@@ -0,0 +1,255 @@
+    public static final String CHANNEL_SENSOR_TEMP = "temperature";
+    public static final String CHANNEL_SENSOR_HUM = "humidity";
+    public static final String CHANNEL_SENSOR_LUX = "lux";
+    public static final String CHANNEL_SENSOR_FLOOD = "flood";
+    public static final String CHANNEL_SENSOR_MOTION = "motion";
+    public static final String CHANNEL_SENSOR_CHARGER = "charger";
+    public static final String CHANNEL_SENSOR_LASTUPDATE = "last_update";
+
+    public static final String CHANNEL_GROUP_SENSE_CONTROL = "control";
+    public static final String CHANNEL_SENSE_KEY = "key";
+
+    public static final String CHANNEL_GROUP_BATTERY = "battery";
+    public static final String CHANNEL_SENSOR_BAT_LEVEL = "batteryLevel";
+    public static final String CHANNEL_SENSOR_BAT_LOW = "lowBattery";
+    public static final String CHANNEL_SENSOR_BAT_VOLT = "voltage";
+
+    public static final String CHANNEL_GROUP_LIGHT_CONTROL = "control";
+    public static final String CHANNEL_LIGHT_COLOR_MODE = "mode";
+    public static final String CHANNEL_LIGHT_POWER = "power";
+    public static final String CHANNEL_LIGHT_DEFSTATE = "defaultState";
+    public static final String CHANNEL_GROUP_LIGHT_CHANNEL = "channel";
+
+    // Bulb/RGBW2 in color mode
+    public static final String CHANNEL_GROUP_COLOR_CONTROL = "color";
+    public static final String CHANNEL_COLOR_PICKER = "hsb";
+    public static final String CHANNEL_COLOR_FULL = "full";
+    public static final String CHANNEL_COLOR_RED = "red";
+    public static final String CHANNEL_COLOR_GREEN = "green";
+    public static final String CHANNEL_COLOR_BLUE = "blue";
+    public static final String CHANNEL_COLOR_WHITE = "white";
+    public static final String CHANNEL_COLOR_GAIN = "gain";
+    public static final String CHANNEL_COLOR_EFFECT = "effect";
+
+    // Bulb/RGBW2 in White Mode
+    public static final String CHANNEL_GROUP_WHITE_CONTROL = "white";
+    public static final String CHANNEL_COLOR_TEMP = "temperature";
+
+    // General
+    public static final String CHANNEL_EVENT_TRIGGER = "event";
+    public static final String CHANNEL_GROUP_SEPARATOR = "#";
+
+    public static final String OPENHAB_HTTP_PORT = "OPENHAB_HTTP_PORT";
+    public static final String OPENHAB_DEF_PORT = "8080";
+
+    // Coap
+    public static int CoIoT_PORT = 5683;
+    public static String COAP_MULTICAST_ADDRESS = "224.0.1.187";
+
+    public static final String COLOIT_URI_BASE = "/cit/";
+    public static final String COLOIT_URI_DEVDESC = COLOIT_URI_BASE + "d";
+    public static final String COLOIT_URI_DEVSTATUS = COLOIT_URI_BASE + "s";
+
+    public static final int COIOT_OPTION_GLOBAL_DEVID = 3332;
+    public static final int COIOT_OPTION_STATUS_VALIDITY = 3412;
+    public static final int COIOT_OPTION_STATUS_SERIAL = 3420;
+
+    public static final byte[] EMPTY_BYTE = new byte[0];
+
+    public static int SHELLY_API_TIMEOUT = 2500;
```

Comment:
```suggestion
    public static int SHELLY_API_TIMEOUT_SECONDS = 2500;
```
?

Reply:
SHELLY_API_TIMEOUT_MS would then be correct